.travis.yml

```diff
@@ -5,5 +5,7 @@ node_js:
   - "4"
   - "5"
   - "6"
+env:
+  - TZ="Europe/Copenhagen"
```

Comment:
Then let me put in a vote for always printing dates in UTC (and moment instances in their associated time zone, or UTC if none).

:)